src_erl/erlBridge/nerlNetStatem.erl

```diff
@@ -249,7 +269,11 @@ train(cast, Param, State) ->
 predict(cast, {sample,CSVname, BatchID, SampleListPredict}, State = #nerlNetStatem_state{features = Features, modelId = ModelId}) ->
   ChunkSizePred = round(length(SampleListPredict)/Features),
   CurrPID = self(),
-  _Pid = spawn(fun()-> erlModule:predict2double(SampleListPredict,ChunkSizePred,Features,ModelId,CurrPID, CSVname, BatchID) end),
+    % ^^^^^^^^^^^^^^^^^^
+   ModelID = 586000901,
```

Comment:
Pay attention to ModelID number - it should be generated and be unique one 
@kapelnik @evgenyan95

Reply:
What about this ModelID line?